### DISCUSSION TITLE
Scripts or contributions at private repositories should be considered at statistics.

### DISCUSSION THREAD

**Discussion by mdnurulmomen** [anuraghazra/github-readme-stats#3063] [Q&A] [answered]
### Is your feature request related to a problem? Please describe.

Statistics depend on public repositories only ! Contributions at private projects are completely ignored. When someone might have many contributions at private repository and github itself has feature to count those contributions.

### Describe the solution you'd like

There should be a feature to consider private contributions and scripts. 

### Describe alternatives you've considered

Either consider private repositories along with public ones, or should be a feature to enable or disabling the option. At least, a declaration could be shown like this statistics only depend on public repositories.

**Comment by qwerty541** [comment, marked as answer]
As noticed in [documentation](https://github.com/anuraghazra/github-readme-stats#github-stats-card) you are able to show private stats on the card after [deploying own Vercel instance](https://github.com/anuraghazra/github-readme-stats#on-vercel) using your own GitHub Personal Access Token (PAT). Public instance uses different token, so it unable to see your private contributions.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/a48c9642-8165-4730-a4b6-1a97f440ec4e)

**Comment by AllAroundD** [comment]
I deployed my own Vercel instance using my own PAT but it still didn't include my private contributions.

**Reply by qwerty541** [reply]
Did you set correct permissions for your PAT? Make sure that you enabled `repo` and `user` permissions.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/9c94bb06-a59d-402e-8014-5f821046b630)

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/c835b969-9850-4cc9-82ed-b9207214b47f)

**Comment by FaizanKamal7** [comment]
@mdnurulmomen I have gave all correct permissions to PAT.And create my personal access token and deployed the forked repository as shown [here ](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own). 
Replaced stats domain with deployed forked repo domain.
But I'm still unable to show private repos contributions in Github Stats.
What am I missing?
